libs/transactions/src/lib/model/tracker-events.ts

```diff
@@ -0,0 +1,9 @@
+import { ScreenViewTrackerEvent } from '@backbase/foundation-ang/observability';
+
+export class TransactionListTrackerEvent extends ScreenViewTrackerEvent {
+  readonly name = 'transaction-list';
+}
+
+export class TransactionDetailsTrackerEvent extends ScreenViewTrackerEvent {
+  readonly name = 'transaction-details';
+}
```

Comment:
Don't forget to add created classes to `index.ts` to make them a part of the public API